packages/fxa-content-server/app/styles/tailwind/flows.css

```diff
@@ -3,10 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .card {
-  @apply relative text-center w-full mobileLandscape:w-120 mobileLandscape:bg-white rounded-xl px-8 pb-10 pt-28 mobileLandscape:pt-20;
+  @apply relative text-center w-full mobileLandscape:w-120 mobileLandscape:bg-white rounded-xl px-8 pb-10 pt-[124px] mobileLandscape:pt-20;
```

Comment:
Heh I think I must've been wrong here on the `12px` guess or I didn't add adequate padding between the logo and the heading the first time. The logo looks great, I see it at `32px` down the page, but there's `32px` space between the logo and the header element and the designs in Figma show `40px` between those and I know we're keeping desktop and mobile designs pretty similar. 

I think this would need to be `pt-33` to equal `132px`, which we probably should just add to the Tailwind config. Not sure if we'll need it anywhere else though because the pseudo element is sorta unique, so one could make the argument that `pt-[8.5rem]` (I think we can use rems with brackets) here works too. 🤷‍♀️

Reply:
I think this would make the second place we use custom sizes, per this goofy <kbd>grep</kbd>:

```sh
git grep -EPIn "\b\-\[.*?\]" packages | cat

packages/fxa-content-server/app/styles/tailwind/ctas.css:14:  @apply -mt-[6px] mx-auto;
packages/fxa-content-server/app/styles/tailwind/flows.css:6:  @apply relative text-center w-full mobileLandscape:w-120 mobileLandscape:bg-white rounded-xl px-8 pb-10 pt-[124px] mobileLandscape:pt-20;
```

Per https://tailwindcss.com/docs/padding, `pt-32` would give us `padding-top: 8rem; /* 128px */`. Not sure if that 4 pixels is "close enough", or if we just add it to the [theme](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/configs/tailwind.js).

As for the [wholly unrelated] `-mt-[6px]` margin, the [docs](https://tailwindcss.com/docs/margin#class-reference) seem to imply we could use something like `-mt-1.5` (which should theoretically convert to`margin-top: -0.375rem; /* -6px */`).

Reply:
Yeah I personally prefer that we just add custom values to the configs. If we wanted something unique _just for content-server_ we could modify it in content-server's config but I feel like for spacing and whatever it should be fine in the global config. However, the TW docs do say something like "if it doesn't make sense to add this to your theme you can use a one-off value" so one could argue it "doesn't make sense to add it to the theme" 🤷‍♀️